src/test/java/org/breedinginsight/api/v1/controller/ProgramControllerIntegrationTest.java

```diff
@@ -2282,6 +2281,22 @@ public void searchProgramUsers() {
         assertEquals(1, pagination.get("currentPage").getAsInt(), "Wrong current page");
     }
 
+    @Test
+    public void getObservationLevels() {
+        Flowable<HttpResponse<String>> call = client.exchange(
+                GET("/programs/" + otherProgram.getId() + "/observation-level")
```

Comment:
need to add an `s` at the end of the path -> `/observation-levels`

Reply:
changed and pushed